contact/views.py

```diff
@@ -0,0 +1,85 @@
+import http
+import requests
+
+from django.conf import settings
+from django.shortcuts import render
+from django.views.generic.base import TemplateView
+from django.views.generic.edit import FormView
+
+from .forms import ContactForm
+
+
+class ContactView(FormView):
+    template_name = 'contact/contact.html'
+    form_class = ContactForm
+    success_url = '/contact/thank-you/'
+
+    def get_context_data(self, **kwargs):
+        """
+        Adds recaptcha_site_key to context for Contact Us page
+        """
+        context = super(ContactView, self).get_context_data(**kwargs)
+        context['recaptcha_site_key'] = settings.RECAPTCHA_SITE_KEY
```

Comment:
These settings should go in `website/settings/production.py` and be read from `os.environ`.

Note: We don't use `local.py` in production; we use environment variables for configuration.

Reply:
Ah, I just saw that you added these in `website/settings/base.py`, which works, but the variables still need to be set-able via envvars. See https://github.com/CenterForOpenScience/cos.io/pull/927/files#r212453359 .